Felienne/hedy
# [FIX] Turtle hidden after program containing 'ask'

**Description**
Currently when we run a program retrieving user input (by using 'ask') the program runs correctly. However, if the next program contains the turtle the turtle stays hidden. This is due to a `hide()` function on the turtle canvas to prevent overlay of the turtle on the user input when both are used in the same program. In this PR we solve this issue of a hidden turtle by checking if the program has a turtle and make sure the canvas is shown (even if already visible, just to be sure). 

**Fix for**
This PR fixes #1427 

**How to test**
First run a program containing user input but not a turtle (!). Next run a program only containing a turtle. The turtle should be visible and execute correctly.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
